### ISSUE TITLE
why not use role=math?

### ISSUE THREAD

**Issue by kimberlysuazo** [mathjax/MathJax-a11y#213]
SSB-BART conducted an audit that turned MathJax up as not using role=math, and flagged it as a severity 5 issue because screen readers with special tooling for math content would not recognize the equation as math. It seems like this was perhaps intentional on MathJax's part. If so, what was the reasoning behind that? And if not, could we switch the role to math?

**Comment by pkra** [comment]
This is flagged incorrectly, most likely due to a misunderstanding of how `role=math` works.

`role=math` behaves like `role=img`: it hides the entire subtree below an element from the accessibility tree. This means, `role=math` only allows you to apply static alttext at the top level (via `aria-label`). 

This is the intentional design specified [in the spec](https://www.w3.org/TR/wai-aria/roles#math) and makes the role effectively useless. For example, the example from the spec where a MathML fragment is wrapped in a div with `role=math` should not work at all, i.e. the the MathML fragment should not be exposed to the accessibility tree. 

In particular, no screen reader should be picking up any content from anything below a `role=math`. If they do, something is not working correctly (e.g., browsers are exposing content in the a11y tree that they should not or screenreaders are hacking the DOM to extract content).

To be fair, the role stems from a time where math was rendered as binary images so it made sense back then; then again, the role has never been updated and is poorly supported across browsers, OS and AT.

Long story short, since the MathJax accessibility extensions provide much more than static alttext,  `role=math` cannot (and should not) be used as it would prevent the extensions from providing rich voicing, navigation and exploration features.

**Comment by kimberlysuazo** [comment]
Thanks for the thorough explanation. My own understanding of ARIA roles is limited as Im just starting to get more familiar with them. It makes sense that `role=math` is not adequate in this case

**Comment by pkra** [comment]
Glad this helped. No worries about starting out -- when it comes to math on the web, everyone is, I'm afraid.
